tests/monitoring/metrics.go

```diff
@@ -23,12 +23,15 @@ import (
 	"context"
 	"time"
 
+	"kubevirt.io/kubevirt/tests/libstorage"
```

Comment:
please move this import in the proper section below

Reply:
fixed